corehq/apps/commtrack/dbaccessors/__init__.py

```diff
@@ -16,22 +16,11 @@ def get_supply_point_ids_in_domain_by_location(domain):
     }
 
 
-def get_supply_points_json_in_domain_by_location(domain):
-    from corehq.apps.commtrack.models import SupplyPointCase
-    results = SupplyPointCase.get_db().view(
-        'supply_point_by_loc/view',
-        startkey=[domain],
-        endkey=[domain, {}],
-        include_docs=True,
-    )
-
-    for result in results:
-        location_id = result['key'][-1]
-        case = result['doc']
-        yield location_id, case
-
-
-def get_supply_point_case_by_location_id(domain, location_id):
+def get_supply_point_by_location_id(domain, location_id):
+    """
+    This also returns closed supply points.
+    Please use location.linked_supply_point() instead.
+    """
```

Comment:
I'll later make a second PR changing the view to also emit case status, which should correct this issue.